### PR TITLE
refactor: simplify ScriptOpCodes.opCodeNameMap

### PR DESCRIPTION
**Pull request by matthewleon**
Use the Guava `BiMap` interface to make the relationship between `opCodeMap` and `opCodeNameMap` explicit.

### CODE REVIEW

File: core/src/main/java/org/bitcoinj/script/ScriptOpCodes.java [hunk 1]
```diff
@@ -421,4 +314,5 @@ public static int getOpCode(String opCodeName) {
 
         return OP_INVALIDOPCODE;
     }
+
```
Comment (schildbach) [inline core/src/main/java/org/bitcoinj/script/ScriptOpCodes.java]: Unrelated change.

**Comment by matthewleon** [comment]
Removed the unrelated change :)

**Comment by schildbach** [comment]
Merged.
